products/compute/terraform.yaml

```diff
@@ -401,6 +401,8 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           [google_compute_image data source](/docs/providers/google/d/datasource_compute_image.html).
           For instance, the image `centos-6-v20180104` includes its family name `centos-6`.
           These images can be referred by family name here.
+      DiskEncryptionKey.rawKey: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
this needs to be lowercase camel, but once that's done it should be ok!

Reply:
Thanks for the quick look and good catch, fixed it in 219cfe6. 

CCLA should be signed soonish as well as soon as I got added to the group from our team.